dev/devicelab/lib/framework/utils.dart

```diff
@@ -188,11 +188,13 @@ Future<Process> startProcess(
   List<String> arguments, {
   Map<String, String> environment,
   String workingDirectory,
+  bool isBot: true, // set to false to pretend not to be on a bot (e.g. to test user-facing outputs)
```

Comment:
Let's add proper Dartdoc to this method instead.

Reply:
done